python/google/protobuf/pyext/descriptor.cc

```diff
@@ -196,6 +196,10 @@ bool Reparse(
   from.SerializeToString(&serialized);
   io::CodedInputStream input(
       reinterpret_cast<const uint8*>(serialized.c_str()), serialized.size());
+  if (cmessage::InternalGetAllowOversizeProtos()) {
+    input.SetTotalBytesLimit(INT_MAX, INT_MAX);
+    input.SetRecursionLimit(INT_MAX);
+  }
```

Comment:
This code is executed when parsing *Descriptor options*, which are not usually that large because they appear in .proto source files. Why did you modify this part?

Reply:
Instances of `CodedInputStream` only appear to be constructed in two places of the codebase. I didn't have a great way of getting a stack trace, but this was ultimately the fix that stopped my use case from crashing.